sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -136,19 +138,23 @@ def __init__(self, hostname, auth=None, **kwargs):
         self._cbs_authenticator = None
         self._auth_timeout = kwargs.pop("auth_timeout", DEFAULT_AUTH_TIMEOUT)
         self._mgmt_links = {}
-        self._retry_policy = kwargs.pop("retry_policy", RetryPolicy())
+        self._retry_policy = retry_policy if retry_policy else RetryPolicy()
+        self._keep_alive_interval = int(keep_alive_interval) if keep_alive_interval else 0
 
         # Connection settings
         self._max_frame_size = kwargs.pop('max_frame_size', None) or MAX_FRAME_SIZE_BYTES
         self._channel_max = kwargs.pop('channel_max', None) or 65535
         self._idle_timeout = kwargs.pop('idle_timeout', None)
         self._properties = kwargs.pop('properties', None)
-        self._network_trace = kwargs.pop("network_trace", False)
+        self._remote_idle_timeout_empty_frame_send_ratio = kwargs.pop(
+        'remote_idle_timeout_empty_frame_send_ratio', None)
+        self._network_trace = debug
```

Comment:
I'm in favour is just using `network_trace` rather than `debug` - as this a better description for what's being activated with this setting.

Reply:
Okay dokey!